webplugin/pluginOptimizer.js

```diff
@@ -245,8 +282,10 @@ const generateBuildFiles = () => {
             }
             var mckApp = data.replace(
                 'KOMMUNICATE_MIN_JS',
-                `"${BUILD_URL}/kommunicate.${version}.min.js"`
-            );
+                // dest is diff for dev and build
+                `"${!env ? BUILD_URL : MCK_CONTEXT_PATH}/kommunicate.${version}.min.js"`
```

Comment:
put this to some variable so that same condition is not required at multiple places 
!env ? BUILD_URL : MCK_CONTEXT_PATH

Reply:
updated the code